src/components/form/Form.view.js

```diff
@@ -8,7 +8,9 @@ import Button from './components/button/Button.view';
 import classes from './form.module.scss';
 
 // ToDo create component that wraps label and textInput
-const Form = ({ onSubmit, preview }) => (
+const Form = ({
+  onSubmit, preview, onChange,
+}) => (
```

Comment:
jak są już trzy to w nowej linii poproszę. I segregowanie w w wcześniej ustalonej kolejności (bool, number, string, itp.)

Reply:
Poprawione